### ISSUE TITLE
About the Input and the Output

### ISSUE THREAD

**Issue by zzhnobug** [AndMu/Wikiled.Sentiment#9]
I have just tried to use this tool to extract sentiment from twitter dataset, but I'm not familiar with c#. And I don't know what the input format is required for this tool, now I have the original csv file twitter_clean_text.csv from your dataset, how can I use this file to do sentiment analysis specifically?
Thanks.

**Comment by AndMu** [comment]
Depends which version you are using.
In the latest it should be
```
Wikiled.Sentiment.ConsoleApp.exe test -All=twitter_clean_text.csv  -Out=twitter_clean_text_out.csv 
```
But because twitter_clean_text is multicolumn CSV, it won't work properly as it expects to be text only.
I would recommend converting it to either XML or JSON array :
```
[{
"Id": "Test", 
"Text": "Test",
...
}]
```

**Comment by zzhnobug** [comment]
I use the latest version 3.3.31. I have tried with twitter_clean_text.001.csv and extracted the two columns : 'Id' and 'Text'. And  I also convert it to JSON file in two ways: orient='index' and orient='columns'. However, When I use the command you suggested, I get three kinds of errors:
'Failed Error parsing Infinity value. Path '', line 1, position 1.' for the original file containing two columns; 'Failed Expected Sentiment type but got: Positive' for orient='columns' JSON file and 'Failed Expected a start of array, got: StartObject' for orient='index' JSON file.

**Comment by AndMu** [comment]
Sorry for an incomplete example. This is tested sample
```
Wikiled.Sentiment.ConsoleApp.exe  test -All=C:\2\input.json -Out=C:\2\result
```
And Json:
```
{
"Neutral": 
	[{
	"Id": "Test1", 
	"Text": "Test",
	},
	{
	"Id": "Test2", 
	"Text": "Test",
	}
	]
}
```

I will put back support to CSV in next version

**Comment by AndMu** [comment]
Published latest v3.5.7 release which will support original CSV.

**Comment by AndMu** [comment]
You can also use free sentiment service
https://github.com/AndMu/Wikiled.Sentiment.Service/tree/master/src/Python.Client
Python code sample:
```
reviews = ['I love this hello kitty decal! I like that the bow is pink instead of red. Only bad thing is that after putting it on the window there a few air bubbles, but that most likely my fault. Shipped fast too.',
                  'I bought this for my 3 yr old daughter when I took it out the pack it had a bad odour, cute but very cheap material easy to ripe.  When I tried it on her it was too big, but of course she liked it so I kept it. I dressed her up in it and she looked cute.']

user_name = socket.gethostname()
host = 'sentiment2.wikiled.com'
port=80
with SentimentConnection(host=host, port=port, client_id=user_name) as connection:
    analysis = SentimentAnalysis(connection, domain='market')
    for result in analysis.detect_sentiment_text(amazon_reviews):
        if result['Stars'] is None:
            print('No Sentinent')
        else:
            print(f'Sentinment Stars: {result["Stars"]:1.2f}')
```
